### PR TITLE
Reworked text and formatting

### PR DESCRIPTION
**Pull request by pronego-mls**

### CODE REVIEW

File: meta/pronego/contao-monoslideshow/de.yml [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
-        (derzeitiger Sponsor) wurde die Erweiterung 2023 nach vergeblichen Kontaktbemühungen zu dem Programmierer 
-        von [Dr. Manuel Lamotte-Schubert/PRONEGO](https://www.pronego.com) übernommen und für Contao 5 angepasst.
+        Sie wurde ursprünglich von Daniel Ritter (https://www.designr.ch)
+        entwickelt und bis ca. 2013 gewartet. Auf Wunsch von Lothar Schwalm (https://die-schreibmaus.de, derzeitiger Sponsor) 
+        wurde die Erweiterung 2023 nach vergeblichen Kontaktbemühungen zu dem Programmierer 
+        von Dr. Manuel Lamotte-Schubert/PRONEGO (https://www.pronego.com) übernommen und für Contao 5 angepasst.
         
-        Die [Monoslideshow](https://monoslideshow.com/) funktioniert nur in Verbindung mit einer Javascript-Datei, 
-        die bis 2023 auf der [Website](https://monoslideshow.com/) erworben werden konnte (Kosten: damals ca. 25 €). 
+        Die Monoslideshow funktioniert nur in Verbindung mit einer Javascript-Datei,
```
Comment (xchs) [inline meta/pronego/contao-monoslideshow/de.yml:12]:
```suggestion
        Die Monoslideshow funktioniert nur in Verbindung mit einer JavaScript-Datei, 
```

File: meta/pronego/contao-monoslideshow/de.yml [hunk 2]
```diff
@@ -1,27 +1,27 @@
...
         Mittlerweile hat der Entwickler der Monoslideshow, Wimer Hazenberg, das Projekt jedoch eingestellt.
-        Man kann ihn aber über seine [Website](https://monokai.com/) kontaktieren und für eine Monoslideshow-Lizenz anfragen.
+        Man kann ihn aber über seine Website https://monokai.com kontaktieren und für eine Monoslideshow-Lizenz anfragen.
 
         Wer bereits eine Monoslideshow-Lizenz besitzt, kann sie mit dieser Erweiterung in Contao 5 
         weiterhin benutzen.
         
         Wer sich die wunderbaren Effekte der Monoslideshow einmal in Live-Galerien ansehen möchte, 
-        bevor er/sie sich die Mühe macht, den Kauf der Javascript-Datei bei [Wimer Hazenberg](https://monokai.com) 
+        bevor er/sie sich die Mühe macht, den Kauf der Javascript-Datei bei Wimer Hazenberg (https://monokai.com)
```
Comment (xchs) [inline meta/pronego/contao-monoslideshow/de.yml:21]:
```suggestion
        bevor er/sie sich die Mühe macht, den Kauf der JavaScript-Datei bei Wimer Hazenberg (https://monokai.com) 
```

File: linter/allowlists/en.txt
Comment (xchs) [inline linter/allowlists/en.txt:1]: You could also add the proper names to the default allowlist: https://github.com/contao/package-metadata/blob/main/linter/allowlists/default.txt

Comment (pronego-mls) [reply linter/allowlists/en.txt:1]: Thanks for the suggestion. I have now moved the names to the default file and replaced Javascript with the proper name JavaScript (with uppercase S).

**Review by xchs — APPROVED** [review]

**Pull request merged**